windows/sound.cpp

```diff
@@ -520,6 +520,11 @@ CSound::CSound ( void           (*fpNewCallback) ( CVector<int16_t>& psData, voi
     // init pointer to our sound object
     pSound = this;
 
+    // According to the docs, driverInfo.asioVersion and driverInfo.sysRef

```

Comment:
If we're going to zero the whole structure, `LoadAndInitializeDriver` should be doing this just before
```
if ( ASIOInit ( &driverInfo ) != ASE_OK )
```
rather than here.

Reply:
Okay. (I thought it would be a smaller change to zero it once, but I guess it shouldn't make a difference)

Reply:
Does it work moved..? :)

Reply:
Works for me...